client/engine/chainservice/eth_chainservice.go

```diff
@@ -75,7 +83,8 @@ func (ecs *EthChainService) SendTransaction(tx protocols.ChainTransaction) *ethT
 		nitroVariablePart := NitroAdjudicator.ConvertVariablePart(state.VariablePart())
 		nitroSignatures := []NitroAdjudicator.IForceMoveSignature{NitroAdjudicator.ConvertSignature(signatures[0]), NitroAdjudicator.ConvertSignature(signatures[1])}
 
-		ethTx, err := ecs.na.ConcludeAndTransferAllAssets(&txOpt, nitroFixedPart, nitroVariablePart, 1, []uint8{0, 0}, nitroSignatures)
+		txOpts := ecs.defaultTxOpts()
```

Comment:
You could return a `bind.TransactOpts` from defaultTxOpts and then you could inline this function call into L87? Just an idea.

Reply:
Good call! 8436395c